src/Views/Appearance.vala

```diff
@@ -485,4 +503,18 @@ public class PantheonShell.Appearance : Gtk.Grid {
 
         return time_double;
     }
+
+    private enum AccentColor {
```

Comment:
Usually we put enums at the top of the class with other properties and variables that are scoped to the class